.github/workflows/auto-release.yml

```diff
@@ -46,7 +46,15 @@ jobs:
       #     wait-interval: 60  # seconds
 
       - name: Setup Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+      - name: Install pypa/build
+        run: >-
+          python3 -m
+          pip install
+          build
+          --user
```

Comment:
TIL something new about the ecosystem today! 

Does this have to be multi-line, btw? Just curious, it looks more like a single-line bash command.

Reply:
i just did a copy and paste, but it's yaml, so it should just fit into one line